resources/js/methods.js

```diff
@@ -0,0 +1,19 @@
+import Emitter from 'tiny-emitter';
+const emitter = new Emitter();
```

Comment:
What's this?

Reply:
See https://v3.vuejs.org/guide/migration/events-api.html#overview